tools/ocp_version_update/update_default_release_versions_to_latest.py

```diff
@@ -48,6 +50,10 @@
 ASSISTED_SERVICE_UPSTREAM_URL = f"https://github.com/{ASSISTED_SERVICE_GITHUB_REPO}.git"
 ASSISTED_SERVICE_MASTER_DEFAULT_OCP_VERSIONS_JSON_URL = \
     f"https://raw.githubusercontent.com/{ASSISTED_SERVICE_GITHUB_REPO}/master/data/default_ocp_versions.json"
+ASSISTED_SERVICE_MASTER_DEFAULT_OS_IMAGES_JSON_URL = \
+    f"https://raw.githubusercontent.com/{ASSISTED_SERVICE_GITHUB_REPO}/master/data/default_os_images.json"
+ASSISTED_SERVICE_MASTER_DEFAULT_RELEASE_IMAGES_JSON_URL = \
+    f"https://raw.githubusercontent.com/{ASSISTED_SERVICE_GITHUB_REPO}/master/data/default_release_images.json"
```

Comment:
Have the prefix as a constant and compound the full path using `DEFAULT_VERSIONS_FILES`, `DEFAULT_OS_IMAGES_FILE`, and `DEFAULT_RELEASE_IMAGES_FILE`

Reply:
done